<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QMainWindow" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientSocket</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>661</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Informations Client</string>
    </property>
    <widget class="QLineEdit" name="NomOrdinateurEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NomUtilisateuEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ArchitectureProcesseurEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="OsEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="NomOrdinateurButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom Ordinateur Distant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NomUtilisateurbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom utilisateur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ArchitectureProcesseurButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Architecture processeur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OS ordinateur</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>651</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Connexion Serveur</string>
    </property>
    <widget class="QLineEdit" name="AdresseIpEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NumeroPortEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="EtatConnexionEdit">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="QuitterButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ConnexionButton">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>160</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse Ip du Serveur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de port du serveur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat de la connexion</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QuitterButton</sender>
   <signal>clicked()</signal>
   <receiver>ClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
